index.html

```diff
@@ -36,6 +36,16 @@ <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
           <input type="submit" value="Rechercher" />
         </form>
       </section>
+      <section id="filters">
+        <h4>Filtres</h4>
```

Comment:
Why h4 ?

Reply:
to differentiate between the filters' title and the text below ("Plus de 500...")

Reply:
Yes but you haven't any h2 or h3 yet

Reply:
actually I hesitated between h3 and h4, filters title has the same font-size as the footer's headings that we put on h4, though I'm not sure it'a a good criterion...
I was thinking of using h2 for other main sections titles ("Hébergement...", "Activités..." etc)

Reply:
ok, changed to h2, new commit arrives